public/ticketmanager.php

```diff
@@ -680,7 +680,7 @@
                     echo "<span>";
                     $msgPayload = "Hi [user=$reportedBy], I'm contacting you about ticket retroachievements.org/ticketmanager.php?i=$ticketID ";
                     $msgPayload = rawurlencode($msgPayload);
-                    echo "<a href='createmessage.php?t=$reportedBy&amp;s=Bug%20Report%20($gameTitle)&p=$msgPayload'>Contact the reporter - $reportedBy</a>";
+                    echo "<a href='createmessage.php?t=$reportedBy&amp;s=Bug%20Report%20(".addcslashes($gameTitle,"'").")&p=$msgPayload'>Contact the reporter - $reportedBy</a>";
```

Comment:
maybe it should be URL encoded since this is in the context of an `href` attribute? for example, any `'` character in the `$gameTitle` should be replaced with `%27`

Reply:
https://www.php.net/rawurlencode should be able to do the trick